sources/osgGA/Manipulator.js

```diff
@@ -40,37 +40,41 @@ Manipulator.prototype = {
         var type = overrideStrat !== undefined ? overrideStrat : this._boundStrategy;
 
         if (type & Manipulator.COMPUTE_HOME_USING_BBOX) {
-            var bs = new BoundingSphere();
-
-            var bb = null;
+            var box;
             if (this._computeBoundNodeMaskOverride === ~0x0) {
-                bb = node.getBoundingBox();
+                box = node.getBoundingBox();
             } else {
                 var ComputeBoundsVisitor = require('osg/ComputeBoundsVisitor').default;
                 var cbv = new ComputeBoundsVisitor();
                 cbv.setNodeMaskOverride(this._computeBoundNodeMaskOverride);
                 cbv.reset();
 
                 cbv.apply(node);
-                bb = cbv.getBoundingBox();
+                box = cbv.getBoundingBox();
             }
 
-            if (bb.valid()) bs.expandByBoundingBox(bb);
+            if (!box.valid()) return node.getBoundingSphere();
 
             // minimum between sphere and box
             if (type & Manipulator.COMPUTE_HOME_USING_SPHERE) {
-                var boundSphere = node.getBound();
-                if (boundSphere.radius() < bs.radius()) return boundSphere;
+                var bsphere = node.getBoundingSphere();
+                if (bsphere.valid() && bsphere.volume() < box.volume()) {
+                    return bsphere;
+                }
             }
 
-            return bs;
+            // return box
```

Comment:
strange comment ?
// returns box as a sphere

function name is not that clear that it must return a boundingsphere, perhaps add it either in name (getHomeBoundingSphere) or function header comment.

Reply:
yes as a sphere

Reply:
removed the comments